src/Model/LabsPost.php

```diff
@@ -36,6 +36,11 @@ public function __construct(
         $this->content = $content;
     }
 
+    public function getIdentifier() : Identifier
+    {
+        return Identifier::labsPost($this->id);
+    }
+
     public function getId() : string
```

Comment:
don't think `getId()` should change as it would remove convenience. Perhaps we can name in a very unambiguous way `getIdentifier()`